### PR TITLE
ENG-1945: Initialize message stream on new attester

### PR DESCRIPTION
**Pull request by alexashley**
w/ @tnishida1 

~We still want to write some tests for this~ 
Edit: tapped out on the integration tests since they would need different setup and tags

### CODE REVIEW

File: api/v1alpha1/condition_types.go [hunk 1]
```diff
@@ -23,10 +23,11 @@ type Condition struct {
...
 
 const (
-	ConditionActive   ConditionType = "Active"
-	ConditionCompiled ConditionType = "Policy"
-	ConditionSecret   ConditionType = "Key"
-	ConditionListener ConditionType = "Listener"
+	ConditionActive      ConditionType = "Active"
+	ConditionCompiled    ConditionType = "Policy"
+	ConditionSecret      ConditionType = "Key"
+	ConditionInitialized ConditionType = "Stream"
```
Comment (alexashley) [inline api/v1alpha1/condition_types.go:29]: Open to suggestions on the naming here.

File: controllers/attester_controller.go [hunk 2]
```diff
@@ -36,9 +37,10 @@ import (
 // AttesterReconciler reconciles a Attester object
 type AttesterReconciler struct {
 	client.Client
-	Log       logr.Logger
-	Scheme    *runtime.Scheme
-	Attesters map[string]attester.Attester
+	Log          logr.Logger
+	Scheme       *runtime.Scheme
+	Attesters    map[string]attester.Attester
+	EventManager eventmanager.EventManager
```
Comment (alexashley) [inline controllers/attester_controller.go:43]: Some plumbing to make the event manager instance available to the attester, but we're weren't sure if there was a better approach here

File: api/v1alpha1/condition_types.go (hunk 1 above)
Comment (ChrisSchreiber) [reply api/v1alpha1/condition_types.go:29]: I think it is less confusing to just use the value in the name `ConditionStream ConditionType = "Stream"`

I also think we could use the same Condition for the Attester and Enforcer resources. So just replace ConditionListener with ConditionStream in the Enforcer Controller

File: controllers/attester_controller.go (hunk 2 above)
Comment (ChrisSchreiber) [reply controllers/attester_controller.go:43]: Yep I think that make sense and is consistent with the Enforcer Reconciler

**Review by ChrisSchreiber — CHANGES_REQUESTED** [review]
Looks good. Would like the Attester and Controller to use the same Condition Type for initializing the event stream.

File: api/v1alpha1/condition_types.go (hunk 1 above)
Comment (alexashley) [reply api/v1alpha1/condition_types.go]: @ChrisSchreiber makes sense, I updated the enforcer controller and its CRDs

**Review by ChrisSchreiber — APPROVED** [review]
👍

**Pull request merged**